test/verify/check-shell-multi-machine

```diff
@@ -372,6 +373,36 @@ class TestMultiMachine(testlib.MachineCase):
         self.allow_hostkey_messages()
         self.allow_journal_messages(".* Failed to resolve hostname bad .*")
 
+        # Check whether AllowMultiHost blocks access already in cockpit-ws.
+
+        self.enable_multihost(m)
+        self.setup_ssh_auth()
```

Comment:
Both of these are already done in setUp(). Should hopefully not hurt, though.

Reply:
Hmm, setup_ssh_auth was necessary, for some reason.

Reply:
> Both of these are already done in setUp(). Should hopefully not hurt, though.

setup_ssh_auth is only done in TestMultiMachineAdd, not in  TestMultiMachine.